apiserver/facades/client/application/application.go

```diff
@@ -328,11 +328,6 @@ func (api *APIBase) Deploy(ctx context.Context, args params.ApplicationsDeploy)
 			// TODO(babbageclunk): rework the deploy API so the
 			// resources are created transactionally to avoid needing
```

Comment:
Might as well delete these comments whilst here since pending resources are deprecated. We can always find them in 3.6. We are also technically implementing the the todo as part of the move to DQLite.